docs/eventing/sources/kafka-source/README.md

```diff
@@ -116,7 +106,7 @@ If you are using Strimzi:
                image: gcr.io/knative-releases/knative.dev/eventing/cmd/event_display
      ```
 
-1. Apply the YAML file by running the command:
+2. Apply the YAML file by running the command:
```

Comment:
Markdown / mkdocs should convert the numbers to the correct ones on when the site is published, so we should leave all these as `1. xxxxx` (or update the ones that weren't to `1. xxxxx`